pai-management/doc/hdfs.md

```diff
@@ -91,6 +91,66 @@ curl http://NAME_NODE_ADDRESS:50070/jmx
 Currently OpenPAI management tool doesn't deploy HDFS in a High Availability (HA) fashion. This will be added in a future release.
 For solution about the HA feature please refer [HDFS High Availability](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/HDFSHighAvailabilityWithNFS.html).
 
+# Access HDFS Data
```

Comment:
Hi Zhaoyu, currently pai suggest user use webhdfs access data. Please add web hdfs api of PAI to tell user how to 
add / delete data etc.

https://github.com/Microsoft/pai/blob/1e8db911a4cb115dab3423fe7a261860966128c9/rest-server/test/hdfs.js